src/components/UploadingJobsDisplay/utils/upload.js

```diff
@@ -119,3 +119,29 @@ export const uploadFiles = (
       .catch(onXHRerror);
   });
 };
+
+const processDeleteResponse = response => {
+  const {
+    status,
+    statusText,
+  } = response;
+
+  if (status !== 204) {
+    const error = {
+      status,
+      statusText,
+    };
+
+    throw new Error(error);
+  }
+};
+
+export const deleteFile = (url, headers) => {
```

Comment:
Can we use async/await here?

Reply:
I would keep using as is as this approach is used in other places in this file. Moreover, we have not decided on strictly using async/await syntax.